openstf/pipeline/create_component_forecast.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+import structlog
+
+import pandas as pd
+
+import openstf.postprocessing.postprocessing as postprocessing
+
+
+def create_components_forecast_pipeline(pj, input_data, weather_data, split_coefs):
+    """Pipeline for creating a component forecast
+
+    Args:
+        pj (dict): Prediction job
+        input_data (pd.DataFrame): Input data fore the components forecast, this usually is a regular forecast
+        weather_data (pd.DataFrame): Weather data with 'radiation' and 'windspeed_100m' columns
+        split_coefs (dict): coefieicnts for the splitting that are determined earlier
+
+    Returns:
+        pd.DataFrame with component forecasts
+
+    """
+    logger = structlog.get_logger(__name__)
+    logger.info("Make components prediction", prediction_id=pj["id"])
+
+    # Make component forecasts
+    try:
+        forecasts = postprocessing.split_forecast_in_components(
+            input_data, weather_data, split_coefs
+        )
+    except Exception as e:
+        # In case something goes wrong we fall back on aan empty dataframe
+        logger.warning(
+            f"Could not make component forecasts: {e}, falling back on series of zeros!",
+            exc_info=e,
+        )
+        forecasts = pd.DataFrame()
+    else:
+
+        forecasts = forecasts.drop(
```

Comment:
How come that column exist if it should not? Would it not make more sense that the `split_forecast_in_components` function does this?

Reply:
Stdev is one of the columns in the dataframe that is returned when loading the forecast inline 35